packages/create-cli/package.json

```diff
@@ -62,6 +61,7 @@
     "@types/ws": "8.5.3",
     "config": "3.3.8",
     "jest": "29.3.1",
+    "rimraf": "5.0.1",
```

Comment:
Why do we have rimraf in the PR?

Reply:
@umutuzgur , I just took the advantage to include the missing devDependency. It doesn't affect any functionality, just let you run the `npm run prepare` under `create-cli` without install all monorepo packages. I can remove it if you prefer.

Reply:
We can keep it for this one 👍